src/utils/set-global-state/state-informant.tsx

```diff
@@ -4,6 +4,8 @@ import { useGlobal } from 'reactn'
 import React from 'react'
 import { render } from '@testing-library/react-native'
 
+// TODO: this probably belongs higher up in a packages directory
```

Comment:
Note this and the comment below in the testing enhancement issue. When we consider enhancing the tests, especially the tests for hooks, we should consider whether this can be used effectively there.

Reply:
Recorded in open issue: https://github.com/js-jslog/harpnative/issues/28#issuecomment-667724767